src/renderers/shaders/ShaderChunk/fog_fragment.glsl.js

```diff
@@ -3,7 +3,7 @@ export default /* glsl */`
 
 	#ifdef FOG_EXP2
 
-		float fogFactor = 1.0 - exp( - fogDensity * fogDensity * fogDepth * fogDepth );
+		float fogFactor = 1.0 - exp( - pow2( fogDensity * fogDepth) );
```

Comment:
```glsl
float fogFactor = 1.0 - exp( - pow2( fogDensity * fogDepth ) );
```
For clarity, can you please fix the formatting and squash the commits?

Reply:
I have fixed the formatting, but don't know how to squash the commits. I made the edit in the online editor.

Reply:
I failed to squash the commits, but achieved the intended outcome by using the sledgehammer of force-push. 😨  Looks right, although the "Thank you, @WestLangley!" disappeared. Thanks, anyway. 😄